terracumber_config/tf_files/SUSEManager-4.1-build-validation.tf

```diff
@@ -1094,21 +1094,14 @@ module "sles11sp3-terminal" {
   providers = {
     libvirt = libvirt.coruscant
   }
-  source             = "./modules/minion"
+  source             = "./modules/pxe_boot"
   base_configuration = module.base_retail.configuration
-  product_version    = "4.1-released"
   name               = "terminal-sles11sp3"
   image              = "sles11sp4" # This is not a typo
   provider_settings = {
     memory             = 1024
     vcpu               = 1
   }
-  server_configuration = {
```

Comment:
why do we no longer need a server configuration for the proxy the terminal connects to? did we change anything?

Reply:
In HCL, variables are defined for each module. The `pxe_minion` module of Sumaform has less variables than the `minion` module.

And somehow it makes sense: the terminal is a perfectly innocent host that will get violently reformatted and onboarded by the proxy. Before that happens, it knows strictly nothing about any SUSE Manager server.